electron/main/index.ts

```diff
@@ -13,7 +13,7 @@ import { Eos, Teos } from '@bitgo/sdk-coin-eos';
 import { Erc20Token, Eth, Gteth } from '@bitgo/sdk-coin-eth';
 import { Ethw } from '@bitgo/sdk-coin-ethw';
 import { Ltc } from '@bitgo/sdk-coin-ltc';
-import { Near, TNear } from '@bitgo/sdk-coin-near';
+import { Near, TNear } from '@bitgo-beta/sdk-coin-near';
```

Comment:
I don't believe this package is supposed to be used outside of bg-ms @bitgoAaron wdyt?

Reply:
i think it's fine, if the coin isn't in a stable release yet there's no other option.